Terminal.Gui/ConsoleDrivers/NetDriver/NetMainLoop.cs

```diff
@@ -150,24 +128,13 @@ private void NetInputHandler ()
 
             if (_resultQueue.Count == 0)
             {
-                _resultQueue.Enqueue (_netEvents.DequeueInput ());
-            }
+                var result = _netEvents.DequeueInput ();
 
-            try
-            {
-                while (_resultQueue.Count > 0 && _resultQueue.TryPeek (out NetEvents.InputResult? result) && result is null)
```

Comment:
This old code added `null` then had this complex code to try and remove nulls.

Simplified to 1 liner - don't add if its `null` in the first place!